Dockerfile

```diff
@@ -10,9 +10,11 @@ RUN ["/bin/bash", "-c", "/cromwell/docker/install.sh /cromwell"]
 
 # Add Cromwell as a service (it will start when the container starts)
 RUN mkdir /etc/service/cromwell && \
-    cp /cromwell/src/main/resources/reference.conf /etc/cromwell.conf && \
+    cp /cromwell/src/main/resources/application.conf /etc/cromwell.conf && \
     cp /cromwell/docker/run.sh /etc/service/cromwell/run
 
+# FIXME - now need to set the conf file specifically
```

Comment:
I'd forgotten about this. I modified stuff to use application.conf the correct way (via ConfigFactory.load) instead of the local file link. We shouldn't be doing this stuff at all - however when the server loads up on the VM we will need to pass in some credentials. Unclear at this moment the best way to do that - looking into it